test/chisel2.py

```diff
@@ -149,7 +149,10 @@ def do_dns_challenges(client, authzs):
             })).read()
         client.answer_challenge(c, c.response(client.key))
     def cleanup():
-        pass
+        urllib2.urlopen("http://localhost:8055/clear-txt",
```

Comment:
Can you pull this URL up into a top level variable that could be overridden with an ENV var? Similar for `chisel.py`.

(Even if you think it shouldn't be overridable by an env var I think I'd prefer to see it as a top level constant that would be easier to spot & change if required)